bin/build_all

```diff
@@ -35,6 +35,13 @@ docker_dir=$(cd $(dirname $(dirname $0))/docker && pwd)
 # Default args
 LANGUAGES=""
 EXCLUDE=""
+pkgs="
+    signing
+    cli
+    sdk/python
+    rest_api
+    validator
+"
```

Comment:
This add shouldn't be here, the pkgs list has been move to build_python and you update that in a later commit.